content/response-to-reviewers.md

```diff
@@ -0,0 +1,134 @@
+# Response to Reviwers
+
+## Reviewer 1
+
+>The authors review how knowledge graphs can be applied in the context of biomedical applications. I found the review to be well-written, highly organized and informative. I have no substantial suggestions and recommend acceptance.
+
+>Minor suggestions which can be fixed by authors in the editorial process:
+
+> - Table 1: What is the organizing principle behind the order of the databases? Would it be better to order them alphabetically?
+> - "or though the use"?
+> - "into a subphrases"?
+> - Table 3: Maybe define or spell out the Types of Sentences.
+> - "This values within"?
+> - "uses a matrices"?
+> - "other effots"?
+> - "then determines how"?
+> - Figure 5b: Font too small
+> - "goal of recommend safe drugs"
+
+**We would like to thank the reviewer for noticing these typos and we have fixed them in this new revision.**
+
+## Reviewer 2
+
+>The manuscript attempts to review methods for knowledge graphs and their biomedical applications. 
+>The review is too general. 
+>It tries to combine disparate lines of research ranging from dimensionality reduction, manifold learning, to graph community detection and frame it under knowledge graph learning. 
+>Although no established definition of knowledge graph exists, an essential property of a knowledge graph is that it models relations between nodes, which don't apply to dimensionality methods (SVD,PCA,,etc.) and other methods such as node2vec. 
+>Almost all of the discussions in this manuscript deals with methods that don't model relations. 
+>The only part that correctly refers to a class knowledge graph methods is translational distance methods.  
+
+**We appreciate the reviewer's perspective; however, we emphasize that the focus of this review is to discuss the pros and cons of building knowledge graphs and representing them in a low dimensional space.
+This space is intended to capture the essence of a knowledge graph and is dependent on the node and edge structure of a graph.
+Once knowledge graphs are represented in this space, they can no longer be considered "true" knowledge graphs. 
+The methods discussed in our review are focused on how this low dimensional space is constructed and we updated our text to be more explicit on how knowledge graphs are used as input.
+**
```

Comment:
```suggestion
```
I guess these can't be set apart on their own - at least in the diff it was bolding everything that followed too.